src/lib/LBT/LBT.cpp

```diff
@@ -34,17 +37,30 @@ uint32_t ICACHE_RAM_ATTR SpreadingFactorToRSSIvalidDelayUs(SX1280_RadioLoRaSprea
   // SF7 ~218us (60us + SF7 symbol time)
   // SF9 ~218us (Odd one out, measured to same as SF7 wait time)
 
-  switch(SF)
-    {
-      case SX1280_LORA_SF5: return 100;
-      case SX1280_LORA_SF6: return 141;
-      case SX1280_LORA_SF7: return 218;
-      case SX1280_LORA_SF9: return 218;
-      default: return 218; // Values above 100mW are not relevant, default to 100mW threshold
-    }
+  if (radio_type == RADIO_TYPE_SX128x_LORA)
+  {
+    switch(SF)
+      {
+        case SX1280_LORA_SF5: return 100;
+        case SX1280_LORA_SF6: return 141;
+        case SX1280_LORA_SF7: return 218;
+        case SX1280_LORA_SF8: return 218;
+        default: return 218;
+      }
+  }
+  else if (radio_type == RADIO_TYPE_SX128x_FLRC)
+  {
+    // case SX1280_FLRC_BT_1: 
```

Comment:
Should this be SX1280_FLRC_BR_0_650_BW_0_6?

Reply:
No the time isn't dependant on the bandwidth, and I don't think it's dependant on the filter either so it could be the fixed value and remove the comment. I just put the comment there because it was the value in the same column as the LoRa, but after more reading realised it wasn't needed but forgot to remove said commented line!